engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-host-modal-helper.service.js

```diff
@@ -0,0 +1,28 @@
+/**
+* @ngdoc service
+* @name  Bastion.content-hosts.service:ContentHostsModalHelper
+*
+* @description
+*   Helper service that contains functionality common amongst content hosts action modals.
+*/
+
+angular.module('Bastion.content-hosts').service('ContentHostsModalHelper',['$uibModal','$q',
+    function ($uibModal,$q) {
+    this.openModal = function(params){
+            if(params.pane){
+                params.pane.invalidate();
+            }
```

Comment:
In fact it looks like there is going to be a fair amount of linting errors but I'll just let eslint find them instead of commenting on them here.